### PR TITLE
[MINOR][DOC] Add mail list to our communication tools and add community introduction

### PR DESCRIPTION
**Pull request by acezen**
## Proposed changes

as title

## Types of changes

What types of changes does your code introduce to GraphAr?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/alibaba/GraphAr/blob/main/CONTRIBUTING.rst) doc
- [ ] I have signed the CLA
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -168,13 +168,16 @@ Code of Conduct
 
 Help us keep GraphAr open and inclusive. Please read and follow our `Code of Conduct`_.
 
-Community
----------
+Getting involved
```
Comment (lixueclaire) [inline README.rst:171]: Getting Involved

File: README.rst [hunk 2]
```diff
@@ -168,13 +168,16 @@ Code of Conduct
...
+----------------
 
-Join the conversation and help the community.
+Join the conversation and help the community. Even if you do not plan to contribute
+to GraphAr itself or GraphAr integrations in other projects, we'd be happy to have you involved
```
Comment (lixueclaire) [inline README.rst:175]: Add a “." at the end

File: README.rst [hunk 3]
```diff
@@ -168,13 +168,16 @@ Code of Conduct
...
 
-- `Discord`_
-- `Weekly Community Meeting`_
+- Join the mailing list: send an email to `graphar+subscribe@googlegroups.com <mailto:graphar+subscribe@googlegroups.com>`_.
+  Share your ideas and use cases for the project.
+- Follow our activity on `GitHub issues <https://github.com/alibaba/GraphAr/issues>`.
+- Join our `Weekly Community Meeting`_
```
Comment (lixueclaire) [inline README.rst:180]: Add a “." at the end

File: README.rst [hunk 4]
```diff
@@ -168,13 +168,16 @@ Code of Conduct
...
 
-Join the conversation and help the community.
+Join the conversation and help the community. Even if you do not plan to contribute
+to GraphAr itself or GraphAr integrations in other projects, we'd be happy to have you involved
 
-- `Discord`_
-- `Weekly Community Meeting`_
+- Join the mailing list: send an email to `graphar+subscribe@googlegroups.com <mailto:graphar+subscribe@googlegroups.com>`_.
+  Share your ideas and use cases for the project.
+- Follow our activity on `GitHub issues <https://github.com/alibaba/GraphAr/issues>`.
```
Comment (lixueclaire) [inline README.rst:179]: fix this link

**Review by lixueclaire — APPROVED** [review]

File: README.rst (hunk 3 above)
Comment (acezen) [reply README.rst]: fixed

File: README.rst (hunk 4 above)
Comment (acezen) [reply README.rst]: fixed

**Comment by acezen** [comment]
hi @lixueclaire , I add a `community.rst` to introduce our community communication and convertion, could you review that again?

**Comment by lixueclaire** [comment]
> hi @lixueclaire , I add a `community.rst` to introduce our community communication and convertion, could you review that again?

Looks good! I think it would be very useful to developers who want to get involved with our community.

**Pull request merged**